### ISSUE TITLE
Warchief's Command Board missing

### ISSUE THREAD

**Issue by bySenom** [AshamaneProject/AshamaneCore#550]
Warchief's Command Board missing in all Horde cities 

Object:

- [ ] http://www.wowhead.com/object=206109/warchiefs-command-board

**Comment by leviathonbeast** [comment]
it is missing as a spawn but its no longer used in retail. its being replaced by the Adventure Log.

**Comment by Neredun** [comment]
@leviathonbeast but in the patch 7.3.5 blizzard added an option to track the boards. 

 Also, i look for the missing boards in my local and there IS two spawns of them, but for some reason they don't show in-game (although you can spawn them again, and they show normally)

**Comment by Neredun** [comment]
The board's aren't missing anymore. At least on Live Server

**Comment by leviathonbeast** [comment]
i think the live server has slightly different DB to what we got :)

**Comment by leviathonbeast** [comment]
@Neredun you are right about the tracker but for the higher levels it becomes useless they work more for the lower levels to show you a suggested zone to level in. for example when you make an allied race on retail they give you 3 suggestions on that board.
